lib/tasks/docs.rake

```diff
@@ -18,7 +18,7 @@ namespace :docs do
       content = File.read(file)
       content = content.gsub(/.*\n\s+(?=module RuboCop)/, "")
 
-      class_doc = content.match(/(\s+#.*)+/).to_s
+      class_doc = content.match(/(\s+#.*)+(?=\n\s+class)/).to_s
```

Comment:
I don't think I fully understand this PR.
Is it fair to assume that this only change in the task impacted on all the `md` changes present here?

- I see a lot of stuff being dropped from the docs - are them all not wanted?
- I see some typos that make me think some where manually changed

I think we can address this against the latest channel (`rubocop-0-71` at this moment) which will then be include on the next channels going forward.